validation/validate.js

```diff
@@ -0,0 +1,43 @@
+const Ajv = require('ajv');
+const path = require('path');
+const schemaMap = require('./schemaMap.json')
+
+let validateSchema = (input="1", schema="1") => {
+    console.log("validating " + input + " against " + schema);
+    console.log("You got validated");
+}
+
+module.exports.validateSchema = validateSchema;
+
+/**
+ * node validation/validate.js <jsonpath> <schema path>
+ */
+if (require.main === module) {
+    let json=""
+    let schema=""
+    console.log(process.argv.length)
+    switch(process.argv.length){
+        case 2:
+            console.log('validating all')
+            validateSchema();
+            break;
+        case 3:
+            json = process.argv[2];
+            json = json.replace('/', '\\')
+            let n = json.lastIndexOf('\\');
+            let jsonKey = json.substring(n + 1);
```

Comment:
Instead of string processing, please find a function in the fsx (file system) module which will take a path and will give you back the file name.